Makefile

```
@@ -1807,7 +1807,7 @@ $(DOC_TARGETS):
 # "Is Rust available?" target
 PHONY += rustavailable
 rustavailable:
-	$(Q)$(CONFIG_SHELL) $(srctree)/scripts/rust-is-available.sh -v && echo "Rust is available!"
+	$(Q)$(CONFIG_SHELL) $(srctree)/scripts/rust-is-available.sh -v && echo >&2 "Rust is available!" || echo >&2 "Rust is not available! Please, run the rustcheck target" 
```

```suggestion
	$(Q)$(CONFIG_SHELL) $(srctree)/scripts/rust-is-available.sh -v && echo "Rust is available!" || echo "Rust does not seem to be available. Please run the rustcheck target." 
```

Also, please split the line in two to avoid going over the usual length limits.